packages/okta-auth-js/lib/fetch/fetchRequest.js

```diff
@@ -12,6 +12,37 @@
 
 var fetch = require('cross-fetch');
 
+
+function readData(response) {
+  if (response.headers.get('Content-Type') &&
+    response.headers.get('Content-Type').toLowerCase().indexOf('application/json') >= 0) {
+  return response.json()
+    // JSON parse can fail if response is not a valid object
+    .catch(e => {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Comment:
nit: we should avoid using `console.error` as much as possible.   console is not 100% reliable and can break items based on chrome and other browser extensions.  example is when `firebug` removes native console commands and replaces them with their own, which caused issues in the past.